rms/search_engine/clients.py

```diff
@@ -0,0 +1,70 @@
+from abc import ABC
+from typing import Any
+
+import httpx
+from rms.search_engine.models import SearchBody, ScopusSearchResponse
+from rms.settings import Settings
+
+
+class ScopusClient(ABC):
+
+    def __init__(self):
+        self.settings = Settings()
+        self.headers = {
+            "httpAccept": "application/json",
+            "X-ELS-APIKey": self.settings.scopus_api_key
+        }
+        self.scopus_search_api_endpoint = self.settings.scopus_search_endpoint
+
+
+class ScopusArticleSearchApi(ScopusClient):
```

Comment:
One stylistic comment regarding the class.
Please order the methods in such a way (`__init__`, public methods, private methods, other magic methods)